### PR TITLE
feat(ci): More aggressive timeouts for integration tests

### PR DESCRIPTION
**Pull request by tibetiroka**
**CI/CD/Testing**

This PR makes it easier to debug integration test failures by configuring their timeout.

## Summary
Our (non-debug) integration tests pass in less than10 seconds consistently. Their timeout is 1500 seconds however, which makes for a really long wait if they get hanged up on something.

This PR changes the timeout to 15 seconds, which shouldn't fail well-behaved tests, but should immediately alert us when things go wrong. It also allows for executing the remaining tests without running into github's runner timeouts.

## Testing Done
I tested that the tests now compute the timeout to 15 seconds.

## Performance Impact
Nothing for passing tests, and massively faster progression for hanging tests.

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
The default is 1500 seconds?

**Comment by tibetiroka** [comment]
Yes. A bit too much imo.

**Review by TheGiraffe3 — APPROVED** [review]

File: tests/integration/IntegrationTests.cmake [hunk 1]
```diff
@@ -27,6 +27,7 @@ foreach(test ${INTEGRATION_TESTS_LIST})
 		set(SET_TEST_PROPS
 	"set_tests_properties([==[${test}]==] PROPERTIES
 		WORKING_DIRECTORY \"${CMAKE_CURRENT_SOURCE_DIR}\"
+		TIMEOUT 15
```
Comment (petervdmeer) [inline tests/integration/IntegrationTests.cmake:30]:
```suggestion
		TIMEOUT 60
```

If tests finish in 10 seconds on a typical day, then I like to have a bit of safety margin for a bad day.

Comment (petervdmeer) [reply tests/integration/IntegrationTests.cmake:30]: I do agree that 1500 seconds is too long.

Comment (petervdmeer) [reply tests/integration/IntegrationTests.cmake:30]: 30 seconds might also be okay.

Comment (tibetiroka) [reply tests/integration/IntegrationTests.cmake:30]:
```suggestion
		TIMEOUT 30
```

**Review by petervdmeer — APPROVED** [review]

**Pull request merged**